### ISSUE TITLE
Certificate Has Expired

### ISSUE THREAD

**Issue by ClockworkWolf** [callaa/Drawpile#342]
Starting today (July 15th) I've been having trouble connecting to any session hosted on drawpile.net, receiving a 'Could not connect to server. The certificate has expired' error. However, on Sunday (July 10th) there was no issue. 

Running Drawpile through a terminal yields:

```
unparseable address: "drawpile.net"
SSL error: 6 "The certificate has expired"
Login failed: "The certificate has expired\n"
Socket error: "The certificate is self-signed, and untrusted"
```
This has been checked using Debian testing (drawpile versions 1.0.2 and 1.0.3), Windows 7 (1.0.2) and Ubuntu 15.10 (1.0.2) with similar symptoms. Of the regular group I draw with, no one else has had this issue today.

A workaround is to host the session locally but then every other member gets the 'expired certificate' error.

No know changes have been made to the router or OS before encountering and system has since been updated and problems have persisted through several restarts, a router reset and fresh compile. Certificates should be up to date and clock is set correctly (though the router time is 1 hour off). No issues involved with internet browsing and streaming.

**Comment by callaa** [comment]
I renewed the server certificate (this time for the next 10 years.) The change won't take effect, however, until the server becomes idle again and restarts.

When the new certificate is loaded, you will see a warning "server certificate has changed" when connecting the first time. If the client doesn't let you pass, you must go to *Preferences->Known hosts* and remove the old certificate for drawpile.net.

I'll close this issue when I've confirmed that the server has restarted and works again.
